android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java

```diff
@@ -1633,28 +1634,12 @@ public static void deleteLexicalModel(Context context, int position, boolean sil
   }
 
   /**
-   * setBannerOptions - Update KMW whether to generate predictions.
-   *                    For now, also display banner
-   * @param mayPredict - boolean whether KMW should generate predictions
-   * @return boolean - Success
-   */
-  public static boolean setBannerOptions(boolean mayPredict) {
-    String url = KMString.format("setBannerOptions(%s)", mayPredict);
-    if (InAppKeyboard != null) {
-      InAppKeyboard.loadJavascript(url);
-    }
-
-    if (SystemKeyboard != null) {
-      SystemKeyboard.loadJavascript(url);
-    }
-
-    return true;
-  }
-
-  /**
-   * setBannerOptions - Update KMW for inapp/system keyboard whether to generate predictions.
-   *                    For now, also display banner
-   * @param mayPredict - boolean whether KMW should generate predictions
+   * setBannerOptions - Update KeymanWeb for inapp/system keyboard whether to generate predictions.
```

Comment:
Reading through KMManager, I am finding the terminology confusing. I believe we have a suggestion banner which displays both predictions and corrections. So that makes me think that the generic thing we are controlling is suggestions, and there are two types of suggestions. But the variable that disables and enables lexical models is named mayPredict. And I'm guessing that when it is false, then you would not correct either? At least when you're in a password field I would not expect corrections when predictions are suspended.

I doubt that it is something to consider dealing with now, but this could be confusing for API consumers.